client/src/components/Users/Auth/SignUpCitizen/SignUpCitizen.tsx

```diff
@@ -297,14 +317,16 @@ function SignupCitizen() {
                             height: '240px',
                           },
                         }}
-                        name="city"
-                        renderValue={() => <MenuItem value="">City</MenuItem>}
+                        name="state"
+                        value={formData.state}
                       >
-                        {makeCitySelectOptions('WA')}
+                        {makeStateSelectOptions()}
                       </Select>
                     </Grid>
                     <Grid item xs={6}>
+                      <InputLabel id="city-select">City</InputLabel>
                       <Select
+                        labelId="city-select"
```

Comment:
Let's make this a regular input instead, so we don't have to provide a city for each state.